addons/purchase_stock/models/purchase_order.py

```diff
@@ -257,7 +257,7 @@ def _prepare_picking(self):
             'picking_type_id': self.picking_type_id.id,
             'partner_id': self.partner_id.id,
             'user_id': False,
-            'date': self.date_order,
+            'create_date': self.date_order,
```

Comment:
I detect there is a problem here. ORM overwrites the `create_date`, so `date_order` cannot be saved if we use `create_date`. It seems `date` is not an useless field 🤔
![Selection_3683](https://github.com/user-attachments/assets/fab8e6fc-501f-4f4e-b737-d90c72c721bd)